liquid_node/commands.py

```diff
@@ -450,6 +474,110 @@ def deletecollection(name):
     purge_collection(name)
 
 
+def importcollection(name, database, blobs, index, method='copy'):
+    """Import a single collection extracted collection.
+
+    :param name: name of the collection you want to import
+    :param database: path to the collections' database data folder
+    :param blobs: path to the collections' blob folder
+    :param index: path to the collections' index, as a .tgz archive, named name-index.tgz
+    :param method: can be 'copy', 'move' or  'link'
+    """
+
+    log.info(f'Importing collection {name}')
+    # parsing `method`, which is not interpreted as a kwarg
+    if method != 'copy':
+        _, method = method.split('=', 1)
+        if method not in ['copy', 'move', 'link']:
+            raise RuntimeError(f'unknown method: {method}')
+
+    node_name = name.lower()
+    if name != node_name:
+        log.info(f'Renaming "{name}" to "{node_name}"')
+    collection_path = Path(config.liquid_volumes) / 'collections' / node_name
+    if collection_path.exists():
+        if confirm(f'collection {name} already exists, overwrite (hoover needs to be running)?'):
+            redeploy = True
+            purge_collection(name)
+        else:
+            redeploy = False
+            log.info(f'exiting and not overwriting data for {name}')
+            exit()
+    else:
+        (collection_path).mkdir(parents=True)
+
+    # copy the pg dir
+    database = Path(database).resolve(strict=True)
+    if not (database / 'data' / 'PG_VERSION').exists():
+        raise RuntimeError("database is not a valid Postgres database")
+    pg_src = database
+    pg_dst = collection_path / 'pg'
+    import_dir(pg_src, pg_dst, method=method)
+
+    # copy the blobs dir
+    blobs = Path(blobs).resolve(strict=True)
+    blob_src = blobs
+    blob_dst = collection_path / 'blobs'
+    import_dir(blob_src, blob_dst, method=method)
+
+    # copy or link the index
+    index = Path(index).resolve(strict=True)
+    index_path = collection_path / 'index'
+    index_path.mkdir(parents=True)
+    if method == 'copy':
```

Comment:
Please add a new function `import_from_docker.import_file` and use it here, instead of adding the three-way if here.